A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml

```diff
@@ -975,6 +975,19 @@
             </Requires>
             <WwiseRTPC SimVar="GROUND VELOCITY" Units="KNOTS" Index="0" RTPCName="SIMVAR_GROUND_VELOCITY" />
         </Sound>
+        
+        <Sound WwiseData="true" WwiseEvent="ptu_sound" NodeName="SOUND_ENGINE_LEFT" LocalVar="XMLVAR_PTU_ON" Continuous="false">
+		<Requires SimVar="TURB ENG N1" Units="PERCENT" Index="1">
+			<Range UpperBound="1" />
+		</Requires>
+	    </Sound>
+	
+	    <Sound WwiseData="true" WwiseEvent="ptu_sound" NodeName="SOUND_ENGINE_LEFT" LocalVar="XMLVAR_PTU_ON" Continuous="false">
+		<Requires SimVar="TURB ENG N1" Units="PERCENT" Index="2">
+			<Range UpperBound="1" />
+		</Requires>
+	    </Sound>
```

Comment:
Shouldn't you use both left and right? Or maybe a node in the center?

Reply:
they are not orientation positional, the speaker panning stays the same